test/parameter_test.py

```diff
@@ -678,6 +702,14 @@ def complete(self):
         self.assertTrue(self.run_locally_split('mynamespace.A --mynamespace.A-p 200 --expected 200'))
         self.assertFalse(self.run_locally_split('mynamespace.A --A-p 200 --expected 200'))
```

Comment:
I also would like a test with parsing a value on the command line, like this line. Can you add that please? :)